reg_tests/test_files/airfoilRANSEdgeTrilinos.rst/airfoilRANSEdgeTrilinos.rst.i

```diff
@@ -0,0 +1,220 @@
+        - name: viscosity
+          type: constant
+          value: 9.1875e-06
+
+    boundary_conditions:
+
+    - wall_boundary_condition: bc_wall
+      target_name: Airfoil
+      wall_user_data:
+        velocity: [0,0]
+        use_wall_function: no
+        turbulent_ke: 0.0
+
+
+    - inflow_boundary_condition: bc_inflow
+      target_name: Inlet
+      inflow_user_data:
+        velocity: [14.96346075389736,1.046347106161880]
+        turbulent_ke: 0.095118
+        specific_dissipation_rate: 2266.4
+
+    - inflow_boundary_condition: bc_bottom
+      target_name: Bottom
+      inflow_user_data:
+        velocity: [14.96346075389736,1.046347106161880]
+        turbulent_ke: 0.095118
+        specific_dissipation_rate: 2266.4
+
+    - open_boundary_condition: bc_open
+      target_name: Outlet
+      open_user_data:
+        velocity: [0,0]
+        pressure: 0.0
+        turbulent_ke: 0.095118
+        specific_dissipation_rate: 2266.4
+
+    - open_boundary_condition: bc_top
+      target_name: Top
+      open_user_data:
+        velocity: [0,0]
+        pressure: 0.0
+        turbulent_ke: 0.095118
+        specific_dissipation_rate: 2266.4
+
+    solution_options:
+      name: myOptions
+      turbulence_model: sst
+      projected_timescale_type: momentum_diag_inv #### Use 1/diagA formulation
+
+      options:
+        - hybrid_factor:
+            velocity: 1.0
+            turbulent_ke: 1.0
+            specific_dissipation_rate: 1.0
+
+        - upw_factor:
+            velocity: 1.0
+            turbulent_ke: 0.0
+            specific_dissipation_rate: 0.0
+
+        - alpha_upw:
+            velocity: 1.0
+            turbulent_ke: 1.0
+            specific_dissipation_rate: 1.0
+
+        - noc_correction:
+            pressure: yes
+
+        - limiter:
+            pressure: no
+            velocity: yes
+            turbulent_ke: yes
+            specific_dissipation_rate: yes
+
+        - projected_nodal_gradient:
+            velocity: element
+            pressure: element
+            turbulent_ke: element
+            specific_dissipation_rate: element
+
+        - relaxation_factor:
+            velocity: 0.7
+            pressure: 0.3
+            turbulent_ke: 0.7
+            specific_dissipation_rate: 0.7
+
+    post_processing:
+
+    - type: surface
+      physics: surface_force_and_moment
+      output_file_name: results/forces.dat
+      frequency: 1
+      parameters: [0,0]
+      target_name: Airfoil
+
+    output:
+      output_data_base_name: results/du91w2.e
+      output_frequency: 5
+      output_node_set: no
+      output_variables:
+       - velocity
+       - pressure
+       - turbulent_ke
+       - specific_dissipation_rate
+       - turbulent_viscosity
+       - minimum_distance_to_wall
+
+    restart:
+      restart_time: 100
+
+Time_Integrators:
+  - StandardTimeIntegrator:
+      name: ti_1
+      start_time: 0
+      time_step: 6.666666666666667e-2
+      termination_step_count: 5
```

Comment:
Can you increase this number, to say 10, so that we have a few time steps to look at when we run this on GPU